packages/app/src/specs/TestingTypePromo.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="flex h-full">
+    <Promo
+      :campaign="campaign"
+      medium="Not set up"
+      :instance-id="promoInstanceId"
+      class="m-auto"
```

Comment:
```suggestion
      class="m-auto mt-[32px]"
```

If we set a top margin here we can get a little closer to the Figma spacing between the top bar and the and the promo itself. What do you think - is there some bad side effect of this that I'm missing?

<img width="1448" alt="Screenshot 2023-06-01 at 4 29 32 PM" src="https://github.com/cypress-io/cypress/assets/8340719/8e2f3ec7-dc98-470c-a4f0-b53def85cf43">

Reply:
The issue with the vertical spacing is how the containing div for the promo is interacting with its siblings and parent. There is an `h-full` (i.e. `height: 100%`) on the containing div which makes the div for the promo be the same height as the entire lower right grid cell.  In the branch for the notifications that Adam has been working on, the SpecsList page has been refactored to use an inner grid to control the different sections of that part of the page.  It makes things much easier to work with.

Reply:
I have a suggestion for how to fix this for now and will add a comment to the code.